src/python/grapl-tests-common/grapl_tests_common/clients/graphql_endpoint_client.py

```diff
@@ -17,20 +24,29 @@ class GraphQLException(Exception):
 
 class GraphqlEndpointClient:
     def __init__(self, jwt: str) -> None:
-        hostname = os.environ["GRAPL_GRAPHQL_HOST"]
-        port = os.environ["GRAPL_GRAPHQL_PORT"]
-        self.endpoint = f"http://{hostname}:{port}"
+        port = int(os.environ["GRAPL_HTTP_FRONTEND_PORT"]) if IS_LOCAL else 443
+        protocol = "http" if IS_LOCAL else "https"
+        self.endpoint = (
+            f'{protocol}://{os.environ["GRAPL_API_HOST"]}:{port}/graphQlEndpoint'
+        )
         self.jwt = jwt
 
     def query(
         self, query: str, variables: Optional[Dict[str, Any]] = None
     ) -> Dict[str, Any]:
         resp = requests.post(
-            f"{self.endpoint}/graphQlEndpoint/graphql",
+            f"{self.endpoint}/graphql",
             params={"query": query, "variables": json.dumps(variables or {})},
             cookies={"grapl_jwt": self.jwt},
         )
-        assert resp.status_code == HTTPStatus.OK, resp.json()
+        if resp.status_code != HTTPStatus.OK:
+            resp_str = "\\n".join(resp.iter_lines(decode_unicode=True))
```

Comment:
Because I don't want the actual log lines to have newlines in them (e.g. when the AssertionError on L44 gets logged). It's easy enough to un-escape these upon logs analysis, it's a far more difficult problem--intractable in general--to deal with multi-line logs.